[2025-01-11T07:00:25.059+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-11T07:00:25.065+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dev_data_processing.france_travail_data_extraction manual__2025-01-11T06:59:52.975894+00:00 [queued]>
[2025-01-11T07:00:25.067+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dev_data_processing.france_travail_data_extraction manual__2025-01-11T06:59:52.975894+00:00 [queued]>
[2025-01-11T07:00:25.067+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-11T07:00:25.075+0000] {taskinstance.py:2889} INFO - Executing <Task(DockerOperator): france_travail_data_extraction> on 2025-01-11 06:59:52.975894+00:00
[2025-01-11T07:00:25.105+0000] {standard_task_runner.py:72} INFO - Started process 161432 to run task
[2025-01-11T07:00:25.112+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dev_data_processing', 'france_travail_data_extraction', 'manual__2025-01-11T06:59:52.975894+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/dev_job_market_dag.py', '--cfg-path', '/tmp/tmpsu3ssgko']
[2025-01-11T07:00:25.113+0000] {standard_task_runner.py:105} INFO - Job 91: Subtask france_travail_data_extraction
[2025-01-11T07:00:25.142+0000] {task_command.py:467} INFO - Running <TaskInstance: dev_data_processing.france_travail_data_extraction manual__2025-01-11T06:59:52.975894+00:00 [running]> on host bun
[2025-01-11T07:00:25.178+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dev_data_processing' AIRFLOW_CTX_TASK_ID='france_travail_data_extraction' AIRFLOW_CTX_EXECUTION_DATE='2025-01-11T06:59:52.975894+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-11T06:59:52.975894+00:00'
[2025-01-11T07:00:25.178+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-11T07:00:25.179+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-11T07:00:25.179+0000] {logging_mixin.py:190} INFO - Current task name:france_travail_data_extraction state:running start_date:2025-01-11 07:00:25.065197+00:00
[2025-01-11T07:00:25.179+0000] {logging_mixin.py:190} INFO - Dag name:dev_data_processing and current dag run status:running
[2025-01-11T07:00:25.179+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-11T07:00:25.222+0000] {docker.py:345} INFO - Starting docker container from image aurel_extract_dock:v3
[2025-01-11T07:01:25.831+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to success. Terminating instance.
[2025-01-11T07:01:25.834+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-01-11T07:01:25.840+0000] {process_utils.py:132} INFO - Sending 15 to group 161432. PIDs of all processes in the group: [161432]
[2025-01-11T07:01:25.841+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 161432
[2025-01-11T07:01:25.842+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-01-11T07:01:25.846+0000] {docker.py:505} INFO - Stopping docker container
[2025-01-11T07:01:36.343+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-11T07:01:36.417+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=161432, status='terminated', exitcode=0, started='07:00:24') (161432) terminated with exit code 0
